HW2/P4/driver.py

```diff
@@ -27,6 +27,33 @@ def py_median_3x3(image, iterations=10, num_threads=1):
 
     return tmpA
 
+# the multithread version
+def py_median_3x3_multi(image, iterations=10, num_threads=4):
+    tmpA = image.copy()
+    tmpB = np.empty_like(tmpA)
+    
+    def py_median_3x3_single(tmpA, tmpB, idx):
+        filtering.median_3x3(tmpA, tmpB, idx, num_threads)
```

Comment:
Did you make changes to the filtering file? It looks like your thread just calls this method, not sure how that works